enterprise/ha/src/test/java/org/neo4j/ha/TestPullUpdatesApplied.java

```diff
@@ -152,7 +153,9 @@ public void failed( InstanceId server )
         Thread.sleep( 15000 );
 
         restart( toKill ); // recovery and branching.
-        boolean hasBranchedData = new File( targetDirectory, "branched" ).listFiles().length > 0;
+        File[] branchedFiles = new File( targetDirectory, "branched" ).listFiles();
+        assertNotNull( branchedFiles );
```

Comment:
I understand the NPE possibility but `branchedFiles == null` feels like a completely valid case for this test. This is not super important however.